commands/cluster_commands/install

```diff
@@ -44,12 +50,102 @@ Variables:
   exit 0
 }
 
+
+
+extract_input_params_from_cli() {
```

Comment:
@zahid-iqbal-alpha 
I was hoping, that we can find a more centralized version for this (in common or so)
you can just catch:
--MYVAL=true
and convert it to:
MYVAL=true

or from file similarly, we could just "source" the file to set the vars.

then, we don't need "special" implementations in every command, but would have a central solution...

Would that be possible?

Reply:
I see, the file is already centrally loaded in common 👍